test/thenCatchP.js

```diff
@@ -0,0 +1,17 @@
+import { assert } from 'chai';
+import * as R from 'ramda';
+import * as RA from '../src';
+
+describe('thenP', function() {
```

Comment:
We need definitely more tests here. One test for `onFulfilled`, one for `onRejected` and one for curring test (here is [example](https://github.com/char0n/ramda-adjunct/blob/7e94b534b07e14052cc222b4e8292855e0232504/test/pathOrLazy.js#L73) how to test a function of arity of 3

For inspiration, please have a look at for example at [allP tests](https://github.com/char0n/ramda-adjunct/blob/a56484defec103cd1bcff2f55099b9699d876892/test/allP.js).

Reply:
Will do! I'm out of town this weekend, I might not be able to spend time on until next weekend. Bear with me, but I will circle back and get them added!

Reply:
No problem. Opensource takes time ;]

Reply:
Okay, I pushed the three tests, and added tests for the types. I didn't know you could do tests for TypeScript types like that. Cool stuff!